src/Distribution/Server/Features/BuildReports/BuildReports.hs

```diff
@@ -89,8 +89,8 @@ data PkgBuildReports = PkgBuildReports {
     buildStatus :: !BuildStatus
 } deriving (Eq, Typeable, Show)
 
-data BuildReports = BuildReports {
-    reportsIndex :: !(Map.Map PackageId PkgBuildReports)
+newtype BuildReports = BuildReports {
```

Comment:
Of all the "use newtype" suggestions I only applied those where the unrerlying field was strict, so as not to change strictness properties of any of the single-constructor single-field datatypes.

Reply:
Ok, makes sense. All the newtype changes reverted.